### ISSUE TITLE
Spot_mini does not open on boot

### ISSUE THREAD

**Issue by albertparedandan** [vdesabou/alfred-spotify-mini-player#250]
Spot_mini player using alfred does not automatically start when booting into my mac. is this normal or is there an option to turn it on?

**Comment by vdesabou** [comment]
Hi,

Is Alfred application started when you're booting ? This workflow needs Alfred to be running obviously.

**Comment by albertparedandan** [comment]
yes it is running

**Comment by albertparedandan** [comment]
yes that's exactly what i meant! sorry if i was being vague, but is there a script or any other workaround so i dont have to launch the workflow manually?

**Comment by vdesabou** [comment]
If you call this command:

```
osascript -e 'tell application "Alfred 3" to run trigger "refresh_library" in workflow "com.vdesabou.spotify.mini.player" with argument "test"'
```

It will trigger a refresh library and start the script needed for Now Playing notifications.

Then you can wrap it in a script and add it as a [Launch Deamon](https://medium.com/@fahimhossain_16989/adding-startup-scripts-to-launch-daemon-on-mac-os-x-sierra-10-12-6-7e0318c74de1)
